Hazel/src/Hazel/Debug/Instrumentor.h

```diff
@@ -9,6 +9,64 @@
 
 namespace Hazel {
 
+	namespace InstrumentorUtils {
+
+		template <size_t N>
+		struct ChangeResult
+		{
+			char Data[N];
+		};
+
+		template <size_t N>
+		constexpr auto RemoveCdecl(const char(&expr)[N])
+		{
+			ChangeResult<N> result = {};
```

Comment:
Initializer list is not needed?

```suggestion
			ChangeResult<N> result;
```

Reply:
Changed.